modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala

```diff
@@ -0,0 +1,33 @@
+package coursier.clitests
+
+import utest._
+
+abstract class SetupTests extends TestSuite {
+
+  def launcher: String
+
+  val tests = Tests {
+
+    test("setup") {
+      TestUtil.withTempDir { tempDir =>
+        val homeDir    = os.Path(tempDir, os.pwd)
+        val installDir = homeDir / "bin"
+        val result = os.proc(
+          launcher,
+          "setup",
+          "--yes",
+          "--user-home",
+          homeDir.toString,
+          "--install-dir",
+          installDir.toString
+        ).call()
+        assert(result.exitCode == 0)
+
+        for (app <- List("scala", "sbt", "cs"))
+          assert(os.exists(installDir / app))
```

Comment:
That failed on Windows: https://github.com/coursier/coursier/actions/runs/3940975124/jobs/6742770477

Did I make a mistake in the way I build the path?

Reply:
I think a `.bat` extension is needed on Windows.